src/helpers/getStatusBarText/index.ts

```diff
@@ -1,11 +1,24 @@
+import { workspace, WorkspaceConfiguration } from 'vscode'
 import getFormattingStatus from '../getFormattingStatus'
 
-export const ENABLED_TEXT = 'Formatting: $(check)'
-export const DISABLED_TEXT = 'Formatting: $(x)'
+type StatusBarLabelT = {
+  formattingEnabled: string
+  formattingDisabled: string
+};
 
-const getStatusBarText = () => {
+export const getStatusBarText = (): string => {
   const isFormattingActivated = getFormattingStatus()
 
+  const configuration: WorkspaceConfiguration = workspace.getConfiguration();
+  let statusBarLabel: StatusBarLabelT | undefined
+
+  statusBarLabel = configuration.get<StatusBarLabelT>(
+    'formattingToggle.statusBarLabel'
+  );
```

Comment:
Why not directly
```suggestion
  const statusBarLabel = configuration.get<StatusBarLabelT>(
    'formattingToggle.statusBarLabel'
  );
```

?

Reply:
I had the same thought — if I remember correctly VS Code was throwing spaghetti-underlines about `undefined` not being assignable to type `string`.

I'm curious what you find when this is bundled up!